tls_codec/derive/Cargo.toml

```diff
@@ -14,7 +14,7 @@ rust-version = "1.60"
 proc-macro = true
 
 [dependencies]
-syn = { version = "2", features = ["parsing"] }
+syn = { version = "2", features = ["parsing", "full"] }
```

Comment:
Do we always need `full`? If not, let's only enable it for the `conditional_deserialization` feature.